src/main/resources/.env

```diff
@@ -0,0 +1,9 @@
+#/**
+# * Copyright (c) 2021 Fundacion Jala.
+# * This software is the confidential and proprietary information of Fundacion Jala
+# * ("Confidential Information"). You shall not disclose such Confidential
+# * Information and shall use it only in accordance with the terms of the
+# * license agreement you entered into with Fundacion Jala
+# */
+TOKEN=00D5e000001NGwY!ARIAQOdTuNqbi1NU1_PfZgIVXM05j_Eh1GclIqv7HJy7MuSQA5evo5OreUKVmvOE6RFE7Pnzb9kTTA6badAsBTRi2yyap6Ra
```

Comment:
only for this course we will upload the token and the uri. in client proyects we must not do this.